src/main/scala/org/allenai/ike/IkeKryoRegistrator.scala

```diff
@@ -0,0 +1,53 @@
+package org.allenai.ike
+
+import org.allenai.ike.patterns.NamedPattern
+
+import com.esotericsoftware.kryo.io.{ Input, Output }
+import com.esotericsoftware.kryo.{ Kryo, Serializer }
+import org.apache.spark.serializer.KryoRegistrator
+
+/** Helper to register custom serializers for Some and None. Since Kryo is a Java library, it
+  * doesn't provide default serializers for these classes.
```

Comment:
Chill is the name of the library that provides Kryo serializations for Scala types.